Description: Patches to upstream build system to enable the relevant bits of the
 test suite for execution during build.
Author: James Page <james.page@ubuntu.com>, Thomas Koch <thomas@koch.ro>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -1297,30 +1297,10 @@
     <target name="test-init" depends="jar,compile-test">
         <delete dir="${test.log.dir}" />
         <delete dir="${test.tmp.dir}" />
-        <delete dir="${test.data.upgrade.dir}" />
-        <delete dir="${test.data.invalid.dir}" />
-        <delete dir="${test.data.buffersize.dir}" />
-        <delete dir="${test.data.kerberos.dir}" />
         <delete dir="${test.data.dir}" />
         <mkdir dir="${test.log.dir}" />
         <mkdir dir="${test.tmp.dir}" />
         <mkdir dir="${test.data.dir}" />
-        <mkdir dir="${test.data.upgrade.dir}" />
-        <mkdir dir="${test.data.invalid.dir}" />
-        <copy todir="${test.data.upgrade.dir}">
-            <fileset dir="${basedir}/src/java/test/data/upgrade"/>
-        </copy>
-        <copy todir="${test.data.invalid.dir}">
-            <fileset dir="${basedir}/src/java/test/data/invalidsnap"/>
-        </copy>
-        <mkdir dir="${test.data.buffersize.dir}" />
-        <copy todir="${test.data.buffersize.dir}">
-            <fileset dir="${basedir}/src/java/test/data/buffersize"/>
-        </copy>
-        <mkdir dir="${test.data.kerberos.dir}" />
-        <copy todir="${test.data.kerberos.dir}">
-            <fileset dir="${basedir}/src/java/test/data/kerberos"/>
-        </copy>
     </target>
 
     <condition property="quicktest">
@@ -1373,6 +1353,9 @@
           <batchtest todir="${test.log.dir}" if="fulltest">
             <fileset dir="${test.src.dir}">
               <include name="**/*${test.category}Test.java"/>
+              <exclude name="**/test/UpgradeTest.java"/>
+              <exclude name="**/test/InvalidSnapshotTest.java"/>
+              <exclude name="**/VerGenTest.java"/>
             </fileset>
           </batchtest>
           <batchtest todir="${test.log.dir}" if="testcase">
--- a/src/contrib/build-contrib.xml
+++ b/src/contrib/build-contrib.xml
@@ -133,7 +133,7 @@
   <target name="jar" depends="compile" unless="skip.contrib">
     <echo message="contrib: ${name}"/>
     <jar
-      jarfile="${build.dir}/zookeeper-${version}-${name}.jar"
+      jarfile="${build.dir}/zookeeper-${name}.jar"
       basedir="${build.classes}"      
     />
   </target>
--- a/src/contrib/zooinspector/build.xml
+++ b/src/contrib/zooinspector/build.xml
@@ -20,7 +20,7 @@
 
 
 	<target name="setjarname">
-		<property name="jarname" value="${build.dir}/zookeeper-${version}-${name}.jar" />
+		<property name="jarname" value="${build.dir}/zookeeper-${name}.jar" />
 	</target>
 
 	<target name="init" depends="checkMainCompiled, zookeeperbuildcontrib.init">
@@ -41,7 +41,7 @@
 			<fileset file="${basedir}/lib/log4j.properties" />
 		</copy>
 		<copy todir="${build.dir}/lib">
-			<fileset file="../../../build/zookeeper-3.3.0.jar" />
+			<fileset file="../../../build/zookeeper.jar" />
 		</copy>
 		<copy todir="${build.dir}">
 			<fileset dir="${basedir}" includes="*.*" excludes="build.xml,ivy.xml" />
@@ -55,7 +55,6 @@
 		<jar jarfile="${jarname}">
 			<manifest>
 				<attribute name="Main-Class" value="org.apache.zookeeper.inspector.ZooInspector" />
-				<attribute name="Class-Path" value="lib/log4j-1.2.15.jar lib/TableLayout-20050920.jar lib/zookeeper-3.3.0.jar lib/jToaster-1.0.4.jar lib" />
 				<attribute name="Built-By" value="${user.name}" />
 				<attribute name="Built-At" value="${build.time}" />
 				<attribute name="Built-On" value="${host.name}" />
